fmriprep/workflows/confounds.py

```diff
@@ -39,7 +38,7 @@ def discover_wf(settings, name="ConfoundDiscoverer"):
     dvars.interface.estimated_memory_gb = settings[
                                               "biggest_epi_file_size_gb"] * 3
     # Frame displacement
-    frame_displace = pe.Node(confounds.FramewiseDisplacement(parameter_source="FSL"),
+    frame_displace = pe.Node(confounds.FramewiseDisplacement(parameter_source="SPM"),
```

Comment:
This interface is never fed anything that used to be ITK matrices - so the answer is no.

"FSL" and "SPM" differ only in order of parameters - it should not influence the FD calculations, but since we are conforming all motion parameters to SPM style I did this for consistency.